js/tests/visual/tooltip.html

```diff
@@ -44,7 +44,22 @@
 <script>
 $(function () {
   $('[data-toggle="tooltip"]').tooltip()
+  testTooltipTransitionError()
 })
+
+function testTooltipTransitionError() {
+  // Should throw an error because tooltip is in transition
+  try {
+    $('#btnOne').tooltip('show').tooltip('hide')
+  }
+  catch (e) {
+    console.error(e.message)
+    $('#btnOne').on('shown.bs.tooltip', function () {
+      $('#btnOne').tooltip('hide').off('shown.bs.tooltip')
+    })
+  }
+    alert('No error thrown for : testTooltipTransitionError')
```

Comment:
This will always alert, no matter whether an error was thrown.
You need to keep a boolean variable to track whether an error was thrown, and then put this `alert()` inside an `if` that uses that variable.